.github/workflows/ci.yaml

```diff
@@ -19,11 +19,11 @@ jobs:
     strategy:
       matrix:
         ruby-version:
-        - '3.1'
```

Comment:
Oh nice - I missed this one - my rudimentary "is this an MIQ repo" checked exactly 3.1 and 3.3 (which is a weird combo, and why I thought it was a safe heuristic), but this repo apparently also had 3.4